stubs/requests/requests/__init__.pyi

```diff
@@ -27,13 +38,7 @@ from .models import PreparedRequest as PreparedRequest, Request as Request, Resp
 from .sessions import Session as Session, session as session
 from .status_codes import codes as codes
 
-__title__: Any
-__build__: Any
-__license__: Any
-__copyright__: Any
-__version__: Any
-
-class NullHandler(logging.Handler):
-    def emit(self, record): ...
+charset_normalizer_version: str | None
+chardet_version: str | None
```

Comment:
I don't think we need to include these in the stub. They're just implementation details of some initialisation logic that `requests` does when you import it; they're not really meant for public consumption. Does stubtest complain if you omit them from the stub and `ignore_missing_stub` is set to `False`?

Reply:
Locally or remotely?

This was added because of local errors when running `MYPYPATH=stubs/requests stubtest requests`

Reply:
If stubtest emits errors for these variables, then I think that is a false-positive error that should be solved by adding an allowlist entry when we change the stubtest setting to `ignore_missing_stub = False`